app/models/concerns/hud_concerns/enrollment.rb

```diff
@@ -29,5 +29,13 @@ module HudConcerns::Enrollment
     scope :in_project, ->(project_ids) do
       joins(:project).where(p_t[:id].in(project_ids))
     end
+
+    def self.invalidate_processing!
+      update_all(processed_as: nil, processed_hash: nil)
+    end
```

Comment:
Oh! I totally didn't realize this was a class method.  That's scary! Anyway, the instance method looks good.  Thanks.